notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -4,7 +4,7 @@
       "cell_type": "code",
```

Comment:
<p>Vertex AI uses your projects Compute Engine service account to create custom training jobs by default. If you want to use the default service account, set SERVICE_ACCOUNT to the service account that you want to use. You can create a service account by following these&nbsp;<a href="https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating" target="_blank">instructions</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>